app/services/cfe/submission_manager.rb

```diff
@@ -16,9 +16,11 @@ def call
       CFE::CreateVehiclesService.call(submission)
       CFE::CreatePropertiesService.call(submission)
       CFE::ObtainAssessmentResultService.call(submission)
+      true
     rescue CFE::SubmissionError => e
       submission.error_message = e.message
       submission.fail!
+      false
```

Comment:
should we also log this error to Sentry so we'll get notified?
```ruby
 Raven.capture_exception(e)
```

Reply:
Yep, good idea.  Will add.